Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -407,7 +435,8 @@ private void ThreadLoadPatcherData()
                         AppDataPath =
                             Application.dataPath.Replace("/Assets",
                                 string.Format("/Temp/PatcherApp{0}", EditorAppSecret)),
-                        OverrideLatestVersionId = EditorOverrideLatestVersionId
+                        OverrideLatestVersionId = EditorOverrideLatestVersionId,
+                        LockFilePath = "patcher.lock"
```

Comment:
I think that using lock file in editor is not necessary.

Reply:
Causes any erros? Allows you to test whole program and avoids edge cases (KISS)

Reply:
I see potential problem - while recompiling scripts, Unity stops executing scripts in not defined way. We cannot be sure that file handle will be released which will cause an error while entering play mode second time.

Reply:
@sp0lsh bump

Reply:
@witcher112 pong
More explicit suggestion that lock file is disabled for editor would be better.